tests/conftest.py

```diff
@@ -994,6 +993,47 @@ def postgis_db():
     yield engine
 
 
+@pytest.fixture()
+def postgres_db():
+    """
+    Using docker, you can run a Postgres test - such as test_import_from_non_postgis_db - as follows:
+        docker run -it --rm -d -p 15432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust postgres:latest
+        KART_POSTGRES_URL='postgresql://postgres:@localhost:15432/postgres' pytest -k postgres --pdb -vvs -nauto
+    """
+    if "KART_POSTGRES_URL" not in os.environ:
+        raise pytest.skip(
+            "Postgres tests require configuration - read docstring at conftest.postgres_db"
+        )
+    engine = Db_Postgis.create_engine(os.environ["KART_POSTGRES_URL"])
+    yield engine
+
+
+@pytest.fixture()
+def no_postgis_db(postgres_db):
+    # Check if the user has superuser access to the database
+    is_superuser = False
+    with postgres_db.connect() as conn:
+        result = conn.execute(
+            "SELECT rolsuper FROM pg_roles WHERE rolname = current_user;"
+        )
+        is_superuser = result.scalar()
+
+    if not is_superuser:
+        raise pytest.skip("Requires superuser access to the database")
+
+    with postgres_db.begin() as transaction:
+        try:
+            transaction.execute("DROP EXTENSION IF EXISTS postgis;")
+            # Test connection without postgis support
+            transaction.execute("SELECT postgis_version();")
+        except sqlalchemy.exc.DBAPIError:
+            pass
+        else:
+            raise pytest.skip("Expected PostGIS to be removed but it is still present")
```

Comment:
I'm not sure why this would happen - perhaps better to let it raise an error in this case rather than quietly skip some tests. Something unexpected happened